arabic/passion/passion.md

```diff
@@ -0,0 +1,3 @@
+إن اردت أن تعيش حياة مليئه بالسعادة٫ عليك اولا ان تجد ما تحب عمله في الحياه ٫ لكل منا اهتماماته و هواياته٫ ولذلك يجب ان نستفيد من هذه الاهتمامات لتحويلها إلى هدف في حياتنا
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.